src/components/radios/_radio.scss

```diff
@@ -9,15 +9,11 @@
     box-shadow: inset 0 0 0 3px var(--ons-color-input-bg);
 
     &::after {
+      display: none;
+    }
+    &:focus,
```

Comment:
Once this line is removed for the focus issue we talked about then it looks good to me

Reply:
removed :)

Reply:
Cool think you've managed to add a load of commits from other PRs though

Reply:
very weird! I have just merged main into my branch